### PR TITLE
Add test coverage

### LINKED ISSUE #2181: Increase code coverage to 85%

### PR DESCRIPTION
**Pull request by sanders41**
Relates to #2181

### Code Changes

* Added tests where coverage is missing.

### Steps to Confirm

* [ ] Run the test suite and verify increased coverage (it looks like this won't be possible because code climate isn't currently updating on the branch)

### Pre-Merge Checklist

* [ ] All CI Pipelines Succeeded
* Documentation:
  * [ ] documentation complete, [PR opened in fidesdocs](https://github.com/ethyca/fidesdocs/pulls)
  * [ ] documentation [issue created in fidesdocs](https://github.com/ethyca/fidesdocs/issues/new/choose)
* [ ] Issue Requirements are Met
* [ ] Relevant Follow-Up Issues Created
* [ ] Update `CHANGELOG.md`

### Description Of Changes

_Write some things here about the changes and any potential caveats_

### CODE REVIEW

**Comment by pattisdr** [comment]
Starting review..

**Review by pattisdr — APPROVED** [review]
This looks fine to me, thanks for the added assertions here and added class-based organization.

File: tests/ops/task/test_filter_results.py [hunk 1]
```diff
@@ -1,936 +1,1084 @@
...
+        # remove_empty_containers(results)
+        # assert results == {}
+
+        ## Removing multiple levels of empty arrays and empty dicts
+        # orig = {
+        #    "A": [[{"B": "C", "D": [{"F": {}}, {"G": []}]}, {"B": "D"}, {"B": "G"}]]
+        # }
+        # results = copy.deepcopy(orig)
+        # remove_empty_containers(results)
+        # assert results == {"A": [[{"B": "C"}, {"B": "D"}, {"B": "G"}]]}
```
[the remark this answers is not in the document]
Comment (sanders41) [reply tests/ops/task/test_filter_results.py]: Oops, I had that there to make sure I got all the cases and forgot to remove it. Removed now. Thanks!

**Comment by sanders41** [comment]
Thanks @pattisdr!

**Comment by pattisdr** [comment]
no problem!

**Pull request merged**